tests/unit/App.py

```diff
@@ -212,6 +214,37 @@ def test_app_init(magic, config, logger, env, always_pull_images):
     assert app.start_services.mock.call_count == 1
 
 
+def test_app_get_tmp_dir(app):
+    assert app.get_tmp_dir() == '/tmp/story.app_uuid'
+
+
+def test_app_create_tmp_dir(patch, app):
+    patch.object(pathlib, 'Path')
+    patch.object(app, 'get_tmp_dir')
+
+    # Yes, called twice to ensure the dir is created just once.
```

Comment:
```suggestion
    # create_tmp_dir is called twice to ensure the dir is created just once.
```

You could also add this in the test comment or test name.